### PR TITLE
Decompile cultists

### LINKED ISSUE #1693: Bring TR2 decompilation percentage to 90%

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TR1X/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description

Resolves #1693.

- Venice masked goon
- Knife thrower
- Shotgun boss

### CODE REVIEW

**Comment by aredfan** [comment]
The knife thrower and the shotgun boss both LGTM. There's just one issue with the Venice masked goon.

- [x] If Lara gets close enough to the Venice masked goon, he'll start running around and bumping into her but without attacking. In OG (or dev snapshot), the goon bumps into Lara a few times too but then he'll start shooting at her again.
20241219_221005_Bartolis_Hideout

File: src/tr2/game/objects/creatures/cultist_1.c [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    }
+}
+
+void __cdecl Cultist1_Control(const int16_t item_num)
+{
+    if (!Creature_Activate(item_num)) {
+        return;
+    }
+
+    __int16 v9; // ax
```
Comment (lahm86) [inline src/tr2/game/objects/creatures/cultist_1.c]: This seems like a leftover from porting maybe?

Comment (rr-) [reply src/tr2/game/objects/creatures/cultist_1.c]: Thanks – fixed.

**Comment by rr-** [comment]
> The knife thrower and the shotgun boss both LGTM. There's just one issue with the Venice masked goon.
> 
> * [ ]  If Lara gets close enough to the Venice masked goon, he'll start running around and bumping into her but without attacking. In OG (or dev snapshot), the goon bumps into Lara a few times too but then he'll start shooting at her again.
>   20241219_221005_Bartolis_Hideout

Rebased on top of #2084 and fixed.

**Review by aredfan — APPROVED** [review]

**Review by lahm86 — APPROVED** [review]

**Pull request merged**